### DISCUSSION TITLE
BIOS v1.17

### DISCUSSION THREAD

**Discussion by jyroxx** [CobanRamo/Lenovo-IdeaPad-S340-15IIL#15] [Q&A]
Has anyone the bios update that is needed to install macOS?

**Comment by CobanRamo** [comment]
I don't quite understand you.
We are talking about a Lenovo IdeaPad S340-15IIL IceLake...
and its bios is currently the one here "[cucn28ww_72ww](https://pcsupport.lenovo.com/de/de/products/laptops-and-netbooks/ideapad-s-series-netbooks/s340-15iil/downloads/driver-list/component?name=BIOS)".

**Comment by CobanRamo** [comment]
it seems there is a new one.
Oh, you mean this one?
Version CUCN29WW(V1.18)

If you need the older one, here it is.
Version [CUCN28WW](https://1drv.ms/u/s!Agl4Dz26E6JdwjvTnpIsacPAy2-j?e=IVLe5d)(V1.17)

What's wrong with the new one? 
Why do you want the old one?
What is wrong with the new bios?

**Reply by jyroxx** [reply]
I tried with the last version of the bios but i still have the same issue. The installer is booting but the mouse does not appear. I managed to go trough the installator but when it’s half loaded the pc reboots and gets kernel panics when it tries to continue the installator.

**Reply by CobanRamo** [reply]
What you are describing has nothing to do with BIOS version.
If you have the same hardware you can go into the debug bios and disable only MSR-Lock, then change the DVMT for Graphic. After that you should be able to use my EFi.
The kernel panic reads more like an incompatible NVME.

In any case, you are looking in the wrong area, Bios version has nothing to do with MacOS compatibility, I could install MacOS on this laptop since Bios version CUCN22WW_66WW (v1.12).

**Reply by jyroxx** [reply]
Okay thanks, I’ll try with an other ssd and tell you. I did something wrong for the mouse but now it’s working just fine. I’ll keep you updated. Have a good day!
